.github/workflows/chipyard-full-flow.yml

```diff
@@ -107,6 +107,61 @@ jobs:
           source env.sh
           cd sims/verilator
           make verilog
+      - name: VLSI test
```

Comment:
@abejgonzalez here is the VLSI tutorial flow, it currently passes CI but maybe it's a bit too hacky?